mediapipe/tasks/ios/build_ios_framework.sh

```diff
@@ -165,9 +165,9 @@ function create_framework_archive {
 
   #----- (3) Move the framework to the destination -----
   if [[ "${ARCHIVE_FRAMEWORK}" == true ]]; then
-    local TARGET_DIR="$(realpath "${FRAMEWORK_NAME}")"
-
     # Create the framework archive directory.
+    mkdir -p "${FRAMEWORK_NAME}"
+    local TARGET_DIR="$(realpath "${FRAMEWORK_NAME}")"
```

Comment:
@schmidt-sebastian After my OS update, the realpath command without creating the directory was failing. I have also cleaned the script up a little bit.